decentraland/constants/ActionStatus.js

```diff
@@ -0,0 +1,9 @@
+const DONE = "done";
+const MISSING = "missing";
+const PENDING = "pending";
+
+export default {
+  DONE,
+  MISSING,
+  PENDING,
+};
```

Comment:
I think we'll want to add the marketplace approval here too

Reply:
These are just for icons. Actiions are here: https://github.com/tasitlabs/tasit/pull/287/files/b1bfdfb59e9de91c8975410eb1c015633ffc043e#diff-8ff39d8220cf223758856fca6a1755d0

Reply:
Oh yeah, what I said here doesn’t make any sense haha. Thanks, I’ll resolve this